db/migrate/20160202155708_add_join_token_to_user_groups.rb

```diff
@@ -0,0 +1,5 @@
+class AddJoinTokenToUserGroups < ActiveRecord::Migration
+  def change
+    add_column :user_groups, :join_token, :string
```

Comment:
Chances of clashing are pretty slim i assume. If we ever searching the db for this token then we'll need an index and may as well make it unique.